events/2022-google-summer-of-code/index.markdown

```diff
@@ -35,31 +41,46 @@ MoveIt is planning to participate again in the [Google Summer of Code](https://s
   * What are the next positions of the other currently active robot arms?
 
   Much of the preliminary work is done in [this PR](https://github.com/ros-planning/moveit/pull/2810).
-* Project size: large
+* Expected outcome: MoveIt is capable of supporting simultaneous motion with two arms.
+* Project size: large (350 hours)
+* Difficulty: hard
+* Preferred skills: several years experience in (ROS, MoveIt, C++)
 * Mentor: Henning Kayser
 
 ### Add ability to attach/detach end-effector links
 
 * Add functionality to MoveIt to programmatically attach and detach grippers from a manipulator. See [this issue](https://github.com/ros-planning/moveit2/issues/208) for a more detailed discussion.
-* Project size: large
+* Expected outcome: End-effector tooling can be swapped programmatically.
+* Project size: large (350 hours)
+* Difficulty: hard
+* Preferred skills: several years experience in (ROS, MoveIt, C++)
```

Comment:
```suggestion
* Preferred skills: Robotics, MoveIt, ROS, C++, Python
```

Reply:
I wrote python when thinking about launch files specifying configurations with multiple grippers.  I also think it will be a really hard project.